operator/.bingo/gen-crd-api-reference-docs.sum

```diff
@@ -2,17 +2,52 @@ github.com/ViaQ/gen-crd-api-reference-docs v0.0.2 h1:OVI5SfpuPo/TTANkqiJvNf+qUWE
 github.com/ViaQ/gen-crd-api-reference-docs v0.0.2/go.mod h1:WxpfZG1IGqhtLb0yM4e0Bl7K8VVmg4FAvQkd3x2R660=
 github.com/ViaQ/gen-crd-api-reference-docs v0.0.3 h1:ChSn0sz5GEkVkjart0RlyeHtuI2ahufu3qyvyjksDq0=
 github.com/ViaQ/gen-crd-api-reference-docs v0.0.3/go.mod h1:WxpfZG1IGqhtLb0yM4e0Bl7K8VVmg4FAvQkd3x2R660=
+github.com/ViaQ/gen-crd-api-reference-docs v0.0.5 h1:23NefJALShMhUoae6dsxmicjT1QN/8SUEZOZ08wIAAI=
```

Comment:
There seems to be no change on the `mod` file for this tool anymore. I think we should also revert the changes to this file.